ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt

```diff
@@ -84,3 +91,74 @@ hideall "Liquid Gaol"
 358.4 "Propeller Wind" sync /:Cruise Chaser:4832:/
 360.3 "Gavel" sync /:Brute Justice:483C:/
 363.3 "Final Sentence" sync /:You:483D:/
+370.6 "Photon" sync /:Cruise Chaser:4836:/
+378.7 "Double Rocket Punch" sync /:Brute Justice:4847:/
+386.0 "Super Jump" sync /:Brute Justice:484A:/
+389.0 "Apocalyptic Ray" sync /:Liquid Rage:484C:/
+390.1 "Apocalyptic Ray" sync /:Liquid Rage:484C:/
+391.2 "Apocalyptic Ray" sync /:Liquid Rage:484C:/
+392.3 "Apocalyptic Ray" sync /:Liquid Rage:484C:/
+393.4 "Apocalyptic Ray" sync /:Liquid Rage:484C:/
+399.0 "Whirlwind" sync /:Cruise Chaser:4B4F:/
+407.1 "Whirlwind" sync /:Cruise Chaser:4B4F:/
+415.7 "Final Sentence" sync /:Brute Justice:4856:/
+425.7 "Eternal Darkness Enrage" sync /:Cruise Chaser:483A:/
+
+### Transition 2: Cruise Chaser, Brute Justice and Alexander Prime
+500.0 "Temporal Stasis" sync /:Alexander Prime:485A:/ window 500,0
+503.1 "Surety and Severity" sync /:Liquid Rage:486[13]:/
+505.1 "Alpha Sword" sync /:Cruise Chaser:486B:/
+505.2 "Flamethrower 1" sync /:Brute Justice:486C:/
+506.3 "Alpha Sword" sync /:Cruise Chaser:486B:/
+507.4 "Flamethrower 2" sync /:Brute Justice:486C:/
+507.5 "Alpha Sword" sync /:Cruise Chaser:486B:/
+521.8 "Chastenning Heat" sync /:Alexander Prime:4A80:/
+525.0 "Divine Spear 1" sync /:Alexander Prime:4A82:/
```

Comment:
These look like they overlap too.  Can you comment out these syncs? Also the alpha swords below?

Reply:
Done.